islandora_pdfjs.module

```diff
@@ -144,7 +146,7 @@ function islandora_pdfjs_islandora_pagecmodel_islandora_solr_object_result_alter
       $page_number = $search_results['solr_doc'][$page_number_field_name][0];
 
       if (islandora_object_access(ISLANDORA_VIEW_OBJECTS, islandora_object_load($book_pid))) {
-        $search_results['object_url'] = "islandora/object/$book_pid";
+        $search_results['object_url'] = Url::fromRoute('islandora.view_object', ['object' => $book_pid], ['absolute' => TRUE])->toString();
```

Comment:
I realize that [it's also done upstream](https://github.com/discoverygarden/islandora_solr/blob/47314417087c8b5acc9500490cacbdc1dea9f566/src/IslandoraSolrQueryProcessor.php#L470), but... not sure that the `['absolute' => TRUE]` bit should be necessary... really... almost tempted to suggest making it a `Url` object all the way through, [since we end up recreating it anyhow](https://github.com/discoverygarden/islandora_solr/blob/47314417087c8b5acc9500490cacbdc1dea9f566/theme/theme.inc#L85)... and any code which _does_ want to modify things would similarly have to do things via a `Url` object...

... If `object_url` was a `Url` object, I think we could then do a [`Url::setRouteParameter()`](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Url.php/function/Url%3A%3AsetRouteParameter/8.6.x) here, something like:
```php
$search_results['object_url']->setRouteParameter('object', $book_pid);
```
I think?... shrug?

Reply:
Eh... _does_ indeed seem possible, but ideal refactors to make it happen would have knock-on effects, changing signatures of theme preprocess bits and the like... let's leave as-is, for now...